pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -783,6 +784,9 @@ type PushoverConfig struct {
 	// Notification title.
 	// +optional
 	Title string `json:"title,omitempty"`
+	// time to live for the alert notification
+	// +optional
+	TTL time.Duration `json:"ttl,omitempty"`
```

Comment:
```suggestion
	TTL *Duration `json:"ttl,omitempty"`
```

We have a [custom Duration type](https://github.com/prometheus-operator/prometheus-operator/blob/6e94864406d265774c1572b187e939f3ea5b8a3e/pkg/apis/monitoring/v1/types.go#L41), which has some useful validations on the API level. Could we use this one instead?

Also, we follow k8s API conventions as much as possible, which means we want to use [pointers for optional fields](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required).